docs/onboarding/09-exposing-twins-over-graphql.md

```diff
@@ -0,0 +1,46 @@
+# Chapter 9: Exposing Digital Twins over GraphQL
+
+- [Why GraphQL?](https://www.apollographql.com/docs/intro/benefits/) - Explains the value proposition of GraphQL

```

Comment:
I also found this video really helpful: [GraphQL in 100 seconds](https://www.youtube.com/watch?v=eIQh02xuVw4), perhaps worth linking as a super brief intro to concepts?

Reply:
added